Library/Homebrew/cmd/audit.rb

```diff
@@ -217,7 +217,9 @@ def audit_conflicts
       begin
         Formulary.factory(c.name)
       rescue FormulaUnavailableError
-        problem "Can't find conflicting formula #{c.name.inspect}."
+        unless c.name =~ HOMEBREW_TAP_FORMULA_REGEX
+          problem "Can't find conflicting formula #{c.name.inspect}."
+        end
```

Comment:
I just find that it would be better to write in this way.
```ruby
begin
  Formulary.factory(c.name)
rescue TapFormulaUnavailableError
  # Don't complain about missing cross-tap conflicts.
rescue FormulaUnavailableError
  problem "Can't find conflicting formula #{c.name.inspect}."
end
```